frappe/integrations/doctype/google_maps/google_maps.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe.model.document import Document
+
+class GoogleMaps(Document):
+	pass
```

Comment:
This is just an empty doctype, the related methods should be setup here and called from the delivery tour doctype. There may be other utils in future which may need these methods, so best to add them where they are relevant and call them in delivery tour.

Reply:
Hi @rmehta ! This is noted. I will be making the changes. Thank!